### PR TITLE
Enhance Navbar hover animation

### LINKED ISSUE #10: Navbar hover animation
Hi 👋🏼 @4darsh-Dev Really good project and front-end is really great. I want to contribute to the Front-End aspect. By giving the navbar better CSS animations.

- [X] I have searched the existing issues.

image
**Feature Description**  
- Adding CSS animation to make the Front-End look more interactive and makes UI/UX better for the end-users.
- Color transitions: Changing the background or text color on hover.
- Scale animations: Enlarging or shrinking elements slightly when hovered.
- Underline effects: Adding or animating an underline on hover.
- Sliding effects: Elements or borders sliding in when hovered.

**Use Case**  
- Having small light weight animations will make the UI more interactive and make it more engaging with the users.

**Benefits**
- Adding a hover animation to the navbar improves user experience by providing visual feedback, making the interface more interactive, responsive, and intuitive.

**Priority**  
- Medium  

**Record**  
- [X] I agree to follow this project's Code of Conduct (Required)  
- [X] I'm a GSSOC contributor  
- [X] I want to work on this issue  
- [X] I'm willing to provide further clarification or assistance if needed.

---

Can you assign this issue to me @4darsh-Dev as part of the Wob'24 contributor.

### PR DESCRIPTION
**Pull request by yashksaini-coder**
Hi 👋🏼 @4darsh-Dev, After checking and configuring the codebase came up with the approach 

- Create a new branch so original source code is not affected.
- To add a Navbar.css that includes css animation effects for the Navbar. 
- Kept it separate so as it does not affect other code and cause errors.
- can be customized easily.

Here's what the changes look like:

navbar-enhance

### CODE REVIEW

**Comment by yashksaini-coder** [comment]
This PR fixes #10 
Review the PR, and add labels `wob24` & `Level2`.

File: frontend/src/components/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    justify-content: space-between;
+    padding: 1rem 2rem;
+    background: transparent;
+    font-size: 1.25rem;
+}
+
+.navbar-logo {
+    display: flex;
+    align-items: center;
+    font-size: 2rem;
```
Comment (4darsh-Dev) [inline frontend/src/components/Navbar.css:13]: font is too much bigger, keep it to 1.6rem

File: frontend/src/components/Navbar.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
+.navbar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem 2rem;
+    background: transparent;
+    font-size: 1.25rem;
```
Comment (4darsh-Dev) [inline frontend/src/components/Navbar.css:7]: font is too much bigger, keep it to 1.1rem

**Review by 4darsh-Dev — CHANGES_REQUESTED** [review]
Nice,
Few changes are required adjust them..

File: frontend/src/components/Navbar.css (hunk 2 above)
Comment (yashksaini-coder) [reply frontend/src/components/Navbar.css:7]: changed the `font-size` to 1.1 rem

image

File: frontend/src/components/Navbar.css (hunk 1 above)
Comment (yashksaini-coder) [reply frontend/src/components/Navbar.css:13]: Changed the `font-size` to 1.6rem.
image

**Comment by yashksaini-coder** [comment]
@4darsh-Dev I have changes the font-size to the requested values, and pushed the changes to the `navbar` branch.

now the PR is ready to fix the #10 issue.

**Review by 4darsh-Dev — APPROVED** [review]
nice, 
seems fine

**Pull request merged**